src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -959,3 +959,30 @@
     text: |-
         az openshift wait -g MyResourceGroup -n MyManagedCluster --updated --interval 60 --timeout 1800
 """
+
+helps['openshift monitor'] = """
+type: group
+short-summary: Commands to manage Log Analytics monitoring. Requires "--workspace-id".
+"""
+
+helps['openshift monitor enable'] = """
+type: command
+short-summary: Enable Log Analytics monitoring. Requires "--workspace-id".
+parameters:
+  - name: --workspace-id
```

Comment:
Could you move this to `_param.py`?

Reply:
@MyronFanQiu should I delete this from here? Every AKS command has `parameters` declared in this file too: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/_help.py#L158 (and following commands too) 
I haven't seen section about it in the guideline file

Reply:
we prefer putting it in `_param.py` for help message of each argument.

Reply:
when removing this block from this file, the lint fails:
![Screenshot from 2020-01-21 14-44-59](https://user-images.githubusercontent.com/5200844/72774228-e338d880-3c5d-11ea-84c7-5f4986881dfb.png)


I have added the help as you requested in the `_param.py` file, but I had to restore it here to fix lint issue